ibc-testkit/tests/core/ics04_channel/acknowledgement.rs

```diff
@@ -34,7 +34,12 @@ struct Fixture {
 #[fixture]
 fn fixture() -> Fixture {
     let client_height = Height::new(0, 2).unwrap();
-    let ctx = MockContext::default().with_client(&ClientId::default(), client_height);
+    let ctx = MockContext::default().with_client_config(
+        MockClientConfig::builder()
+            .client_id(ClientId::default())
```

Comment:
Given that we employ a default value for the `ClientId`, can we utilize `#[builder(default(..)]` for the `client_id` field?